src/EnergyPlus/HeatRecovery.cc

```diff
@@ -1442,161 +1464,111 @@ namespace HeatRecovery {
 		using DataHVACGlobals::Heating;
```

Comment:
@rraustad Those warnings typically mean there's a stray using statement that isn't needed.  Looks like these may be the culprit here since blocks of code were moved out?